tests/ui_tests/test_cluster_list_page.py

```diff
@@ -861,6 +889,364 @@ def test_field_tooltips_on_cluster_config_page(self, app_fs, sdk_client_fs):
+        self, sdk_client_fs: ADCMClient, path, app_fs
+    ):
+        """Invisible and advanced for groups true for fields false.
+        In this case no elements presented on page
+        """
+
+        _, cluster_config_page = self.prepare_cluster_and_config(sdk_client_fs, path, app_fs)
+        cluster_config_page.config.check_no_rows_or_groups_on_page()
+        cluster_config_page.config.check_no_rows_or_groups_on_page_with_advanced()
+
+    @parametrize_by_data_subdirs(__file__, "group_advanced_false_invisible_true_field_advanced_true_invisible_true")
+    def test_group_advanced_false_invisible_true_field_advanced_true_invisible_true_on_cluster_config_page(
+        self, sdk_client_fs: ADCMClient, path, app_fs
+    ):
+        """Invisible and advanced for groups true for fields false.
+        In this case no elements presented on page
+        """
+
+        _, cluster_config_page = self.prepare_cluster_and_config(sdk_client_fs, path, app_fs)
+        cluster_config_page.config.check_no_rows_or_groups_on_page()
+        cluster_config_page.config.check_no_rows_or_groups_on_page_with_advanced()
+
+    @parametrize_by_data_subdirs(__file__, "group_advanced_true_invisible_false_field_advanced_false_invisible_false")
+    def test_group_advanced_true_invisible_false_field_advanced_false_invisible_false_on_cluster_config_page(
+        self, sdk_client_fs: ADCMClient, path, app_fs
+    ):
+        """Fields and groups visible only if advanced enabled."""
+
+        _, cluster_config_page = self.prepare_cluster_and_config(sdk_client_fs, path, app_fs)
+        cluster_config_page.config.check_no_rows_or_groups_on_page()
+        cluster_config_page.config.click_on_advanced()
+        with allure.step('Check that 1 field and 1 group is visible'):
+            assert len(cluster_config_page.config.get_group_names()) == 1, "There should be 1 group"
+            assert len(cluster_config_page.config.get_all_config_rows()) == 2, "There should be group and field row"
+
+    @parametrize_by_data_subdirs(__file__, "group_advanced_true_invisible_false_field_advanced_false_invisible_true")
+    def test_group_advanced_true_invisible_false_field_advanced_false_invisible_true_on_cluster_config_page(
+        self, sdk_client_fs: ADCMClient, path, app_fs
+    ):
+        """Invisible=false and advanced=true for groups advanced=false and invisible=true for fields"""
+
+        _, cluster_config_page = self.prepare_cluster_and_config(sdk_client_fs, path, app_fs)
+        cluster_config_page.config.check_no_rows_or_groups_on_page()
+        cluster_config_page.config.click_on_advanced()
+        with allure.step('Check that 1 group is visible and field is not'):
+            assert len(cluster_config_page.config.get_group_names()) == 1, "There should be 1 group"
+            assert len(cluster_config_page.config.get_all_config_rows()) == 1, "There should be group and no field row"
```

Comment:
`get_all_config_rows` return groups and fields? or there's one field in group and none outside of it?

Reply:
it returns both fields and groups